franklin/scripts/scripts/scripts.js

```diff
@@ -45,6 +45,8 @@ const CONFIG = {
  */
 
 const miloLibs = setLibs(LIBS);
+export const { createTag, getConfig } = await import(`${miloLibs}/utils/utils.js`);
```

Comment:
Bringing in the entire `utils` library is expensive, do we need to bring everything in? That's why we just lifted `createTag` and added it to the repo, which can be done with `getConfig` as well.

Reply:
No problem! I'll extract them.

Reply:
I took another look at the scripts.js. The self-invoking loadPage() brings the entire utils library as well. Since the browser caches it, all in all we would always have fetched once. I'll move these lines lower in the file to pay the DNS tax after the style loads and consolidate the imports but I don't think this import comes with extra cost.

Reply:
Oh good catch, didn't notice that from the initial Milo implementation. In that case, I agree, since it's already cached and our Core Web Vitals aren't impacted much, it should be fine to use the functions from the cloud-hosted milo lib.